### DISCUSSION TITLE
Lock scroll on homepage

### DISCUSSION THREAD

**Discussion by meroxdotdev** [squidfunk/mkdocs-material#7148] [Q&A]
Hello, please let me know how can I implement on my first page from mkdocs squid to can t be scrolled from phone when you are at the top or bottom of the page ( like on https://squidfunk.github.io/mkdocs-material/ )

Thank you!

**Comment by alexvoss** [comment]
Not sure what you mean by "can't be scrolled"? I assume you are referring to the parallax effect where different elements scroll at different rates? There question how to implement this has come up a number of times in the discussions, for example here: https://github.com/squidfunk/mkdocs-material/discussions/6253

**Reply by meroxdotdev** [reply]
This is exactly what I m looking for:

![image](https://github.com/squidfunk/mkdocs-material/assets/35839230/37ae2e8b-5618-4edd-b216-fcbac2c1d468)

https://github.com/squidfunk/mkdocs-material/assets/35839230/45bbe943-bd74-4944-9efe-77c5f5a2722a

I dont want the homepage to be scrolled

**Reply by alexvoss** [reply]
Is it the form submission that is the problem? Otherwise, I am not sure why you would want to turn off the refresh the page gesture?m Preventing form re-submissions may have other solutions.
